modules/op-release-notes-1-19-4.adoc

```diff
@@ -0,0 +1,82 @@
+// This module is included in the following assemblies:
+// * release_notes/op-release-notes-1-19.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="op-release-notes-1-19-4_{context}"]
+= Release notes for {pipelines-title} 1.19.4
+
+With this update, {pipelines-title} General Availability (GA) 1.19.4 is available on {OCP} 4.16 and later versions.
+
+[id="fixed-issues-1-19-4_{context}"]
+== Fixed issues
+
+.Pipelines
+
+Git resolver no longer ignores certificates in the cluster custom PKI::
+Before this update, a regression caused the Git resolver to no longer use the OpenShift Proxy custom-configured PKIs. This prevented the Git resolver from resolving references to a self-hosted Git provider. With this update, the full CA bundle configured in the OpenShift Proxy is trusted by the system in all components, including the Git resolver.
```

Comment:
Expand CA to certificate authority (CA) as per the first usage guidelines. Refer to this link for more information: https://redhat-documentation.github.io/supplementary-style-guide/#certificate-authority